CMakeLists.txt

```diff
@@ -191,6 +192,7 @@ option(FLB_IN_HEAD                     "Enable Head input plugin"
 option(FLB_IN_PROC                     "Enable Process input plugin"                  Yes)
 option(FLB_IN_SYSTEMD                  "Enable Systemd input plugin"                  Yes)
 option(FLB_IN_DUMMY                    "Enable Dummy input plugin"                    Yes)
+option(FLB_IN_SNMP                     "Enable SNMP input plugin"                     Yes)
```

Comment:
To confirm - does this work on Windows and macOS as well? It will need disabling if not in their specific override cmake/ files.
- https://github.com/fluent/fluent-bit/blob/master/cmake/macros.cmake
- https://github.com/fluent/fluent-bit/blob/master/cmake/windows-setup.cmake